
# [CVE-2022-22846](https://github.com/paulc/dnslib/issues/30)

## Description

- `The dnslib package through 0.9.16 for Python does not verify that the ID value in a DNS reply matches an ID value in a query.`

## Cvss Data

- **Access Vector**:
  - NETWORK
- **Base Score**:
  - 5.0

## Scores

- **Exploitability Score**:
  - 10.0
- **Impact Score**:
  - 2.9
- **Base Severity**:
  - MEDIUM

## Other Information

- **Publish Date**:
  - 2022-01-10 14:12:58
- **Vulnerability Status**:
  - Analyzed

## References

- **cve@mitre.org**: https://github.com/paulc/dnslib/issues/30
